int8 state
bool init
bool probe_complete
float32 linear_position
bool contact_made
